bruno.js

```diff
@@ -44,11 +48,23 @@ manager.addAnswer('fr', 'serve.cocktail', "C'est parti ! :cocktail:");
 manager.addAnswer('fr', 'serve.cocktail', 'Voilà pour toi ! :cocktail:');
 manager.addAnswer('fr', 'serve.cocktail', 'Tiens, santé ! :cocktail:');
 
+// demande de food
+manager.addDocument('fr', 'tu me prépares des %food%', 'serve.food');
+manager.addDocument('fr', 'je peux te prendre un %food%', 'serve.food');
+manager.addDocument('fr', 'un %food%', 'serve.food');
+manager.addDocument('fr', 'je veux bien un %food%', 'serve.food');
+
+manager.addAnswer('fr', 'serve.food', 'Et voilà :spaghetti: Tu veux du parmesan ?');
+
 // tu as quoi
 manager.addDocument('fr', 'tu as quoi ?', 'what.do.you.serve');
+manager.addDocument('fr', "tu as quoi à manger?", 'what.do.you.serve.food');
 manager.addDocument('fr', 'tu sers quoi ?', 'what.do.you.serve');
+manager.addDocument('fr', "tu sers quoi ? à manger?", 'what.do.you.serve.food');
 manager.addDocument('fr', "qu'est-ce que t'as ?", 'what.do.you.serve');
+manager.addDocument('fr', "qu'est-ce que tu sers à manger?", 'what.do.you.serve.food');
 manager.addDocument('fr', "qu'est-ce que tu sers ?", 'what.do.you.serve');
+manager.addDocument('fr', "qu'est-ce que tu sers à manger?", 'what.do.you.serve.food');
```

Comment:
I see a lot of duplication, can you refactor my whole mess into something maintainable ? otherwise no approval, k thx bye